web/src/components/feature/MoleTable.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { render, screen } from '@testing-library/react'
+import { MoleTable } from './MoleTable'
+import '@testing-library/jest-dom/extend-expect'
+
+const testComponents = {
+  '1': { chemical_formula: 'CO2', alt_name: 'Carbondioxide' },
```

Comment:
Should these be camelCase?

Reply:
Yes, but that is what they look like at the moment coming from the API.